crates/cli-flags/src/lib.rs

```diff
@@ -74,7 +74,7 @@ wasmtime_option_group! {
 
         /// The number of decommits to do per batch. A batch size of 1
         /// effectively disables decommit batching. (default: 1)
-        pub pooling_decommit_batch_size: Option<u32>,
+        pub pooling_decommit_batch_size: Option<usize>,
```

Comment:
This option was added to the struct but never used below: https://github.com/bytecodealliance/wasmtime/blob/main/crates/wasmtime/src/config.rs#L2585-L2588